java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/Properties.java

```diff
@@ -215,6 +216,7 @@ public int getPropertyAsInt(String key) {
      *
      * @param key The property key.
      * @return The property value interpreted as an integer, or the default value.
+     * @throws PropertyException Raised if the property value is not a valid integer.
```

Comment:
Would be good to document what we throw if the key doesn't designate a valid Ice property.